tests/src/system/basic/WskBasicTests.scala

```diff
@@ -477,15 +477,15 @@ class WskBasicTests
                 activation =>
                     activation.response.result shouldBe Some(dynamicParams.toJson)
                     activation.duration shouldBe 0L // shouldn't exist but CLI generates it
-                    activation.end shouldBe Instant.EPOCH.toEpochMilli // shouldn't exist but CLI generates it
+                    activation.end shouldBe Instant.ofEpochMilli(0L) // shouldn't exist but CLI generates it
```

Comment:
Yes, the CLI will stop automatically producing these fields when the schema is removed. Removal of the schema decoding will have to be done for each command and subcommand. For instance, this PR currently removes the decoding for action invocations only. In order to remove mentioned generated fields, trigger fire would have to be updated.